SpeechResponder/HelpWindow.xaml.cs

```diff
@@ -1,4 +1,4 @@
-﻿using System;
+using System;
```

Comment:
Not sure we need this?

Reply:
Nothing changed here. It shouldn't be included.